<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="2" column="0" colspan="2">
      <widget class="QRadioButton" name="raNone">
       <property name="text">
        <string>No Proxy</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QRadioButton" name="raTor">
       <property name="text">
        <string>Tor (SOCKS5)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="laHost">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Host:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="liHost">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="laPort">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="liPort">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>9050</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="liListen">
       <property name="text">
        <string>8444</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="laListen">
       <property name="text">
        <string>Listen port:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>raTor</sender>
   <signal>toggled(bool)</signal>
   <receiver>liHost</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>raTor</sender>
   <signal>toggled(bool)</signal>
   <receiver>liPort</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>raTor</sender>
   <signal>toggled(bool)</signal>
   <receiver>laHost</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>raTor</sender>
   <signal>toggled(bool)</signal>
   <receiver>laPort</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>24</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>raNone</sender>
   <signal>toggled(bool)</signal>
   <receiver>laListen</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>43</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>raNone</sender>
   <signal>toggled(bool)</signal>
   <receiver>liListen</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
